recipe/meta.yaml

```diff
@@ -91,15 +86,15 @@ requirements:
     - readline
     - llvm-openmp  # [osx]
     - libgomp      # [linux and not aarch64]
-    - parmed >=3.4.1,<5.0.0a
+    - parmed >=4.0.0,<5.0.0a0
```

Comment:
@swails - can you confirm which `parmed` version is vendored in Ambertools 23? Thanks!

Reply:
4.0.0 is enough I think!